exercises/05-reselect/src/modules/users/users-selectors.ts

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect'
+
+import { RootState } from 'modules/root/root-reducer'
+import { User } from './user-types'
+
+const getState = (state: RootState) => state.users
+
+export const getTitle = createSelector(getState, (state) => state.title)
+
+const getUsers = createSelector(getState, (state) => state.users)
+
+export const getUserList = createSelector(getUsers, (users) =>
+  users.map((user: User) => ({
```

Comment:
I'd separate this into a function called `withUppercaseLastName` it's easier to read then...
users.map(withUppercaseLastName)

Reply:
Done as proposed, see ff52f60

Reply:
Renamed to `withUpperCaseLastName` (with capitalized `C`) in 1dbafcf, based on review comment for exercise 6.